posthog-android/src/main/java/com/posthog/android/replay/internal/PostHogLogCatIntegration.kt

```diff
@@ -2,21 +2,26 @@ package com.posthog.android.replay.internal
 
 import android.annotation.SuppressLint
 import android.os.Build
+import com.posthog.PostHog
+import com.posthog.PostHogIntegration
 import com.posthog.android.PostHogAndroidConfig
 import com.posthog.internal.interruptSafely
 import com.posthog.internal.replay.RRPluginEvent
 import com.posthog.internal.replay.capture
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-internal class PostHogLogCatWatcher(private val config: PostHogAndroidConfig) {
+internal class PostHogLogCatIntegration(private val config: PostHogAndroidConfig) : PostHogIntegration {
```

Comment:
the logcat integration now is totally decoupled from the app lifecycle and session replay integration, its installed but only captures something if the session is active and replay is enabled